static/schema.json

```diff
@@ -973,14 +970,14 @@
           "description": "Native Mutations.",
           "allOf": [
             {
-              "$ref": "#/definitions/NativeQueries"
+              "$ref": "#/definitions/NativeMutations"
             }
           ]
         }
       }
     },
     "NativeQueries": {
-      "description": "Metadata information of Native Operations.",
+      "description": "Metadata information of Native Queries.",
```

Comment:
Is this a change we _want_ to make? (IIRC the sanctioned name is "Native Operations").

It looks like this file hasn't been updated for some time and has collected a delta.

Reply:
> It looks like this file hasn't been updated for some time and has collected a delta.

You are right.

> Is this a change we want to make? (IIRC the sanctioned name is "Native Operations").

Yes - in config v5, we split native queries to two separate fields and nest them under native operations. This particular bit represents the `queries` field, so it makes sense to call it native queries. The `mutations` field is called native mutations as well, and they both nest under the umbrella term Native Operations:

https://github.com/hasura/ndc-postgres/blob/39d79809d77d25fc0c524f087875852784398232/static/schema.json#L956